themes/bootstrap3/templates/RecordTab/holdingsils/boundwith.phtml

```diff
@@ -0,0 +1,21 @@
+<div class="bound-with-records">
+    <h4>
+        <?=$this->transEsc(
+    $callNumberCount < 2 ? 'bound_with_heading_one_callnumber' : 'bound_with_heading_multiple_callnumbers',
+    ['%%callnumber%%' => $this->escapeHtml($callNumber)]
+)
```

Comment:
I really don't like the way this is (not) indented, but it's what php-cs-fixer insists on.

Reply:
Sometimes it makes bad decisions in templates. You might be able to get better results by leaving the <?= on its own line, then indenting the content on the next line, then closing with ?> below that.

Reply:
I see that my earlier comment got a bit garbled -- that'll teach me to write something in a rush from my phone. What I was trying to suggest was something like this:

```
        <?=
            $this->transEsc(
                $callNumberCount < 2
                  ? 'bound_with_heading_one_callnumber'
                  : 'bound_with_heading_multiple_callnumbers',
                ['%%callnumber%%' => $this->escapeHtml($callNumber)]
            )
        ?>
```

I think with this basic arrangement, php-cs-fixer might tweak the indentation a little bit, but it shouldn't completely de-indent anything in ugly ways.

Reply:
Yes that worked!  Much cleaner.